packages/svelte/src/runtime/store/index.js

```diff
@@ -26,7 +26,7 @@ export function readable(value, start) {
  * Create a `Writable` store that allows both updating and reading by subscription.
  * @template T
  * @param {T} value initial value
```

Comment:
I think this is also optional in `master` if I understand that's what the `=` syntax means. Do we want to do the same here?

https://github.com/sveltejs/svelte/blob/3bc791bcba97f0810165c7a2e215563993a0989b/src/runtime/store/index.ts#L68

Reply:
both ways work, `{string=} foo` is google closure syntax while `{string} [foo]` is from jsdoc itself. ts knows both https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns  I think being consistent is important, using the [] syntax is a bit more to type but closer to jsdoc and allows telling about a default values.